frontend/src/components/stages/allocation_reveal_view.ts

```diff
@@ -62,7 +62,7 @@ export class AllocationReveal extends MobxLitElement {
                   const percentage =
                     participantAnswer.allocationMap[asset.id]?.percentage;
                   return html`<div class="table-cell">
-                    ${percentage?.toFixed(1) ?? 'N/A'}%
+                    ${percentage?.toFixed(1) ?? '0'}%
```

Comment:
This is fine for now; let's consider resolving this in the underlying allocation asset moving forward. @vivtsai FYI.